packages/backend/src/controllers/authentication.ts

```diff
@@ -127,14 +143,27 @@ const createOpenIdUserFromUserInfo = (
     return openIdUser;
 };
 
+/**
+ * Not all possible values are reflected in the underlying PassportProfile type,
+ * at least for AzureAD + openid-client
+ */
+type PassportProfileLike = PassportProfile &
+    Partial<{
+        email: string;
```

Comment:
I'm 99% sure this is openid-client's strategy misbehaving, but opted to not chase it further for now.